src/node_native_module.h

```diff
@@ -3,34 +3,70 @@
 
 #if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
 
-#include "node_internals.h"
+#include <map>
+#include <set>
+#include <string>
+#include "env.h"
+#include "node_union_bytes.h"
+#include "v8.h"
 
 namespace node {
 namespace native_module {
 
-// The native (C++) side of the native module compilation.
+typedef typename std::map<std::string, UnionBytes> NativeModuleRecordMap;
+typedef typename std::map<std::string, std::string> NativeModuleHashMap;
```

Comment:
btw, did you consider using `std::unordered_map` and `std::unordered_set` over `std::map`/`std::set`? I don’t know if we need to worry a lot about the performance tradeoffs here, but I personally tend to use the unordered versions when I can

Reply:
I don't think there is a reason not to use the unordered version...good idea, I'll change that